src/Entity/ElectedRepresentative.php

```diff
@@ -0,0 +1,305 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use DateTime;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use libphonenumber\PhoneNumber;
+use Misd\PhoneNumberBundle\Validator\Constraints\PhoneNumber as AssertPhoneNumber;
+
+/**
+ * @ORM\Entity
+ */
+class ElectedRepresentative
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="IDENTITY")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(length=50)
+     */
+    private $lastName;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(length=50)
+     */
+    private $firstName;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(length=10)
+     */
+    private $gender;
+
+    /**
+     * @var DateTime
+     *
+     * @ORM\Column(type="date")
+     */
+    private $birthDate;
+
+    /**
+     * @var ?string
+     *
+     * @ORM\Column(length=255, nullable=true)
+     */
+    private $birthPlace;
+
+    /**
+     * @var ?int
+     *
+     * @ORM\Column(type="bigint", nullable=true)
+     */
+    private $officialId;
+
+    /**
+     * @var ?string
+     *
+     * @ORM\Column(length=255, nullable=true)
+     */
+    private $contactEmail;
+
+    /**
+     * @var ?string
+     *
+     * @ORM\Column(type="phone_number", nullable=true)
+     *
+     * @AssertPhoneNumber(defaultRegion="FR")
+     */
+    private $contactPhone;
+
+    /**
+     * @var ?string
+     *
+     * @ORM\Column(nullable=true)
+     */
+    private $comment;
+
+    /**
+     * @var ?bool
+     *
+     * @ORM\Column(type="boolean", nullable=true)
+     */
+    private $isSupportingLaREM;
+
+    /**
+     * @var ?bool
+     *
+     * @ORM\Column(type="boolean", nullable=true)
+     */
+    private $hasFollowedTraining;
+
+    /**
+     * @var ?bool
+     *
+     * @ORM\Column(type="boolean", nullable=true)
+     */
+    private $isAdherent;
+
+    /**
+     * @var ?Adherent
+     *
+     * @ORM\OneToOne(targetEntity="AppBundle\Entity\Adherent")
+     * @ORM\JoinColumn(name="adherent_id", referencedColumnName="id", onDelete="SET NULL")
+     */
+    private $adherent;
+
+    /**
+     * @var SocialNetworkLink
+     *
+     * @ORM\OneToMany(
+     *     targetEntity="SocialNetworkLink",
+     *     mappedBy="electedRepresentative",
+     *     cascade={"all"},
+     *     orphanRemoval=true,
+     *     fetch="EAGER"
+     * )
+     */
+    private $socialNetworkLinks;
+
+    public function __construct(
+        string $firstName,
+        string $lastName,
+        string $gender,
+        DateTime $birthDate,
```

Comment:
`\DateTime ...`

Reply:
No need, because this class is added with `use` https://github.com/EnMarche/en-marche.fr/pull/4268/files/f731dc4ba393703422bfc50dd14e55750a25789c#diff-c68d9b46651fc11eee1bd9dd7652c8dbR5